### PR TITLE
Fix issues with `venv` object for tox>=3.8.0 (on py36)

### PR DESCRIPTION
**Pull request by chiqomar**
I don't know if the tests work, but the code is working properly on my machine now, python 3.6.8.

### CODE REVIEW

**Comment by thomaslee** [comment]
Ran into what I believe is the same issue on Python 3.7.3:

```
... snip ...
  File "/Users/tom/.local/share/virtualenvs/tools-YTG8DW9i/lib/python3.7/site-packages/tox_pipenv/plugin.py", line 164, in tox_runenvreport
    pipfile_path = _clone_pipfile(venv)
  File "/Users/tom/.local/share/virtualenvs/tools-YTG8DW9i/lib/python3.7/site-packages/tox_pipenv/plugin.py", line 22, in _clone_pipfile
    root_pipfile_path = venv.session.config.toxinidir.join("Pipfile")
AttributeError: 'VirtualEnv' object has no attribute 'session'
```

Haven't had a chance to verify @chiqomar's attempted fix yet.

File: tox_pipenv/plugin.py [hunk 1]
```diff
@@ -19,17 +22,19 @@ def _init_pipenv_environ():
...
 
     venv_pipfile_path = venv.path.join("Pipfile")
-    if not root_pipfile_path.exists():
+    if not os.path.exists(root_pipfile_path):
         with open(str(root_pipfile_path), "a"):
             os.utime(str(root_pipfile_path), None)
 
     if not venv_pipfile_path.check():
-        root_pipfile_path.copy(venv_pipfile_path)
+        root_pipfile_path = copy(venv_pipfile_path)
```
Comment (oleahy) [inline tox_pipenv/plugin.py:37]: This patch did not work for me,  I got the following error:

 :  
File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 22, in <module>
    from .project import Project, SourceNotFound
  File "/usr/local/lib/python3.7/site-packages/pipenv/project.py", line 110, in <module>
    raise RuntimeError("Given PIPENV_PIPFILE is not found!")
RuntimeError: Given PIPENV_PIPFILE is not found!

Changing line 37 to:
` shutil.copyfile(root_pipfile_path, venv_pipfile_path)`

Seemed to make things work for me, I'm not sure what the intention of the original line is

**Comment by uhurusurfa** [comment]
@tonybaloney @AlmogCohen Any chance this can be reviewed and moved forward? I tested the fix and works fine (minus the line 37 change in plugin.py - the original works for me with tox 3.14)

**Comment by gaborbernat** [comment]
Superseded by #66

**Pull request closed without merging**